openstack-mixin/targets.libsonnet

```diff
@@ -101,7 +101,7 @@ local lokiQuery = g.query.loki;
         'openstack_identity_users{%(queriesSelector)s}' % vars
       )
       + panel.timeSeries.queryOptions.withInterval('1m')
-      + prometheusQuery.withLegendFormat('{{instance}}'),
+      + prometheusQuery.withLegendFormat('{{cluster}}'),
```

Comment:
I broadly disagree with the re-introduction of cluster as a label for this mixin, especially since the README in this version makes no mention of it, but I'm open to hearing arguments. Just know that this was already heavily discussed during the original implementation of this integration.

Reply:
Trying to use this label that won't exist for any user of the old mixin causes the legends to become unformatted.